docs/source/usage.rst

```diff
@@ -83,34 +82,6 @@ In addition, the ``ContextConfig`` and ``Config`` objects can be initialized wit
 - netCDF4/xarray filepath (``str`` or ``Path`` object) or ``Dataset``
 
 
-StreamConfig
-~~~~~~~~~~~~
-A ``StreamConfig`` object defines a specific `ioos_qc` test module and test function along with the configuration parameters in which to run it with.
-
-.. note::
-
-    In earlier versions, ``StreamConfig`` was known as ``QcConfig``.
-
-Usage
-^^^^^
-
-.. code-block:: python
-    :linenos:
-    :caption: A basic ``StreamConfig`` object
-
-    from ioos_qc.config import StreamConfig
-
-    config = {
-        'qartod': {
-            'gross_range_test': {
-                'suspect_span': [1, 11],
-                'fail_span': [0, 12],
-            }
-        }
-    }
-    c = StreamConfig(config)
-
-
 ContextConfig
 ~~~~~~~~~~~~~
 A ``ContextConfig`` object defines multiple ``StreamConfig`` objects as well as optional `region` and `window` objects.
```

Comment:
Should `StreamConfig` be replaced w/ something else here? What's the new object called?

Reply:
ContextConfig can take any ConfigType if I understood the code correctly. I changed to that instead of StreamConfig.

Reply:
Yea I see that now, `ConfigType` object is what we're passing when initializing `ContextConfig` class.